SpatialGDK/Source/SpatialGDK/Private/Utils/RPCContainer.cpp

```diff
@@ -193,11 +205,17 @@ bool FRPCContainer::ApplyFunction(FPendingRPCParams& Params)
 	{
 		return true;
 	}
-	else
-	{
+
 #if !UE_BUILD_SHIPPING
-		LogRPCError(ErrorInfo, QueueType, Params);
+	LogRPCError(ErrorInfo, QueueType, Params);
 #endif
-		return ErrorInfo.bShouldDrop;
+	return ErrorInfo.bShouldDrop;
+}
+
+void FRPCContainer::ExecuteRPCQueueProcessingUpdateFunction(const FPendingRPCParams& Params)
+{
+	if (RPCQueueProcessingUpdateDelegate.IsBound())
```

Comment:
Can we check this? Seems like a bit of a death sentence for the GDK to not have this bound